### PR TITLE
128T-5.5.9 Release Notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.5.md [hunk 1]
```diff
@@ -28,6 +28,135 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49756 RDP sessions failure over peer path:** Resolved an issue that caused RDP traffic to fail when adaptive encryption and AppId are both enabled.
+------
+- **I95-49791 Add audit rules to track modification of grub config files:** Added rules to log notifications in case of changes to grub configuration files.
+------
+- **I95-49925 GRE tunnel health-check not migrating sessions when path is down:** The GRE tunnel manager now removes all sessions before adding new ones rather than modifying the existing sessions.
+------
+- **I95-49974 Stuck flow not cleared when reverse metadata is incomplete:** Resolved an issue where reverse metadata is coming through incomplete - without the source tenant. The source tenant has been added to the reverse metadata.
+------
+- **I95-50014 Hitting Buffer Overflow during configuration changes:** Reolved an issue where a config change request may not make it to a managed router, and returns a buffer overflow error.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:103]: Typo: Resolved
Is this issue in other release notes as well?

File: docs/release_notes_128t_5.5.md [hunk 2]
```diff
@@ -28,6 +28,135 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48931 Service area Highway crash:** Now prevents crashing in SSR's highway process in unusual race conditions when a session's flow is removed before the session is fully established.
+------
+- **I95-48942 Routing policy filter condition reference type not validated:** Added a check to verify that when a routing policy condition references a filter, the condition type and filter type match.
+------
+- **I95-48965, I95-50070, I95-51086 Race condition with routing updates inducing crash in highway process:** Resolved an issue where a routing change that affects the `forwarding-table` can incur a race condition with sessions completing and being removed, which could lead to a highway crash and restart.
+------
+-  **I95-49350 BFD echo generating latency overhead:** BFD echo tests are now staggered to minimize application latency's contribution to overall peer path latency.
+------
+- **I95-49377 Transmit packets dropped by NIC for established sessions - packet counters are incrementing and can be seen in packet capture, but not seen by next-hop:** Added hooks for NIC driver to trigger an unrecoverable event and invoke the Highway lockup detector mechanism.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:77]: This issue is part-in-parcel with I95-50445. They both pertain to MDD. One is the fix, this one is the watchdog. We should clean up all release notes related to this issue so they have a comma separated list of issues. I would remove this item and add a bulleted list item under I95-50445 to indicate
- Added hooks for NIC driver to trigger an unrecoverable event and invoke the Highway lockup detector mechanism

Please do this for all release notes that contain this issue.

File: docs/release_notes_128t_5.5.md [hunk 3]
```diff
@@ -28,6 +28,135 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+[DPDK| -- ] ERROR (00007f03ec18e700) i40e_dev_alarm_handler(): ICR0: malicious programming detected
+[DPDK| -- ] WARN  (00007f03ec18e700) i40e_handle_mdd_event(): Malicious Driver Detection event 0x02 on TX queue 6 PF number 0x01 VF number 0x00 device 0000:08:00.1
+[DPDK| -- ] WARN  (00007f03ec18e700) i40e_handle_mdd_event(): TX driver issue detected on PF
+```
+------
+- **I95-50534 Race condition between NetworkInterfaceManager and FastLane:** Resolved a race condition caused by adding and deleting the same network interface in a very short window of time, potentially causing a system crash.
+------
+- **I95-50554 No dynamic synchronization of repos to the routers:** Resolved an issue where it was necessary to restart 128T on the Conductor in order for the Conductor to recognize newly added repositories and sync them down to the assets. Authenticated repos are now automatically synchronized when repos are added to the conductor.
+------
+- **I95-50722 Highway crashes during SSR migration:** Resolved a crash in the SSR's highway process, due to a race condition between configuration changes and BFD sessions.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:134]: "SSR migration" refers to moving the management of a SSR from conductor-managed to Mist-managed.
This should be "Highway crashes during session migration"

File: docs/release_notes_128t_5.5.md [hunk 4]
```diff
@@ -28,6 +28,135 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-50736 SSH key change not propogated to secondary conductor:** Resolved an issue where an SSH key change to `/etc/128technology/ssh/pdc_ssh_key` was not automatically detected and resynced between peer node and conductor nodes.
+------
+- **I95-50754 Race condition between icmp ping request and a reverse flow:** Resolved a crash due to a race condition when `service ping icmp-request` is matched against a partially installed flow.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:138]: ICMP should be capitalized in the heading

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**